common/containers/Home/index.jsx

```diff
@@ -23,16 +25,41 @@ export class Home extends Component {
     }
   }
 
+  handleNavigateUsers() {
+    if (__CLIENT__) {
+      window.location.href = '/users'
+    }
+  }
+
+  componentDidMount() {
+    this.constructor.fetchData(this.props.dispatch, this.props)
+  }
+
+  static fetchData(dispatch) {
+    dispatch(updateJWT())
+  }
+
   render() {
+    const isAdmin = this.props.isAdmin
     return (
-      <HomeComponent onEditProfile={this.handleEditProfile} onSignOut={this.handleSignOut}/>
+      <HomeComponent onEditProfile={this.handleEditProfile} onSignOut={this.handleSignOut} onNavigateUsers={this.handleNavigateUsers} isAdmin={isAdmin}/>
```

Comment:
As mentioned above, let's give all three of these event handlers more descriptive and consistent naming:
- `onClickUsers`
- `onClickEditProfile`
- `onClickSignOut`

We can also break this onto multiple lines to improve readability:
```js
  <HomeComponent
    onClickUsers={this.handleClickUsers}
    onClickEditProfile={this.handleEditProfile}
    onClickSignOut={this.handleSignOut}
    ...
    />
```